smart_campus/app/models.py

```diff
@@ -1,3 +1,222 @@
 from django.contrib.gis.db import models
+from django.contrib.auth.base_user import (
+    BaseUserManager, AbstractBaseUser
+)
 
-# Create your models here.
+class Beacon(models.Model):
+    beacon_id = models.CharField(max_length=200, primary_key=True)
+    name = models.CharField(max_length=200)
+    description = models.CharField(max_length=200, blank=True)
+    location = models.GeometryField(srid=4326, null=True)
+    owner_group = models.ForeignKey('UserGroup', null=True, on_delete=models.SET_NULL)
+    station = models.ForeignKey('Station', null=True, on_delete=models.SET_NULL)
+
+    def __str__(self):
+        return self.name
+
+class UserManager(BaseUserManager):
+    def create_user(self, email, password=None):
```

Comment:
According to the document. 
The password will be set to an unusable password, which check_password() will never return True
Shall I check if the password is None?
```
if not password:
    raise ValueError('Password is needed.')
```